src/rules/order-imports.ts

```diff
@@ -531,8 +531,8 @@ module.exports = {
 		} catch (error) {
 			// Malformed configuration
 			return {
-				Program: function(node) {
-					context.report(node, error.message);
+				Program: function (node) {
+					context.report({ node, message: error.message });
```

Comment:
Fixes violation of: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/main/docs/rules/no-deprecated-report-api.md